src/app/views/app-sections/AppTitle.tsx

```diff
@@ -28,16 +30,27 @@ export function appTitleDisplayOnFullScreen(
     <div className={classes.graphExplorerLabelContainer} role={'heading'} aria-level={1}>
       {!minimised &&
         <>
-          {displayGraphLabel(classes)}
+          {displayGraphLabel(classes, permissionType)}
         </>}
     </div>
+    <div style={{ marginTop: 15 }}>
```

Comment:
Can we separate the CSS in the .scss files?

Reply:
i think this was added by another member when we were trying out stuff like a toggle so i've removed it because we are no longer using the toggle. but for the future i'll remember to keep css seperate!